tests/test-coreos.sh

```diff
@@ -38,28 +39,36 @@ fetch_repo() {
 
 fetch_latest_rhcos_image() {
   local image_path=$1
-  local image_url="https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/latest"
-  local latest_image_gz=$(curl ${image_url}/ | grep -Po "rhcos-([[:digit:]]).([[:digit:]]){1,2}.([[:digit:]]{1,2})-x86_64-qemu.x86_64.qcow2.gz" | head -1)
-  if [[ -z "${latest_image_gz}" ]]; then
-    echo failed to get the latest image name. check url and regex
+  local installer_image_branch=$2
+  local image_menu_url="https://raw.githubusercontent.com/openshift/installer/${installer_image_branch}/data/data/coreos/rhcos.json"
+  local image_menu="rhcos.json"
+  mkdir -p "${image_path}"
+  wget -nv -O "${image_path}/${image_menu}" "${image_menu_url}"
+
+  local image_gz_url
+  image_gz_url=$(< "${image_path}/${image_menu}" docker run --rm -i stedolan/jq '.architectures.x86_64.artifacts.qemu.formats["qcow2.gz"].disk.location' | tr -d '"')
```

Comment:
Don't we have `jq` already ?

Reply:
wasn't aware we were. I see that the kcli.sh script is using jq, but I didn't want to assume we have it installed (as it's not mentioned on the [setup.sh](https://github.com/RHsyseng/rhcos-slb/blob/simplify-networking/tests/setup.sh) script)

Reply:
I mean the coreos qemu images have jq

Reply:
This code is run by the test environment, meaning whatever gitations is using to run our tests (currently ubuntu)